### PR TITLE
New: Add noscript message

### PR DESCRIPTION
**Pull request by stramel**
Display a message for users with javascript disabled

### CODE REVIEW

**Comment by stramel** [comment]
@abdonrd Can you trigger a rebuild on my branch for me?

**Comment by abdonrd** [comment]
It will continue to fail as long as we don't merge #1003.

**Comment by stramel** [comment]
@robdodson

File: index.html [hunk 1]
```diff
@@ -108,6 +108,9 @@
   </head>
   <body>
     <my-app></my-app>
+    <noscript>
+      Please enable Javascript to view this website.
```
Comment (robdodson) [inline index.html]: mind changing to `JavaScript`? I realize that's super pedantic but... :)

Comment (abdonrd) [inline index.html]: I like it! haha 😄

Comment (stramel) [inline index.html]: Done. I was on the fence of which to use. The coin flip failed me. 😞

**Comment by abdonrd** [comment]
Thanks @stramel! 👏

**Pull request merged**